app/controllers/admin/export/document_controller.rb

```diff
@@ -33,6 +33,7 @@ def migrated
 
     document.editions.each do |edition|
       Whitehall::InternalLinkUpdater.new(edition).call
+      Whitehall::SearchIndex.delete(edition)
```

Comment:
It's likely only the most recent edition of the document will be present in the search index.  Since this is looping through all editions, do you know how `Whitehall::SearchIndex.delete` will behave for an edition that is not present in the search index?

Reply:
Hi Bruce,

We've tested this (via console rails c) and it appears to behave exactly the same if the edition exists or not.  The method call delegates out to a worker job (SearchIndexDeleteWorker) that is queued.

When run the worker job calls `Whitehall::SearchIndex.for(index.to_sym, logger: logger).delete(link)` which will do nothing (return nil) if the edition is not in the index.

Think we're safe!